util/measures/2021/additional-stratifications.json

```diff
@@ -1 +1,2 @@
-[]
```

Comment:
I see some missing stratifier info: CMS74v10, CMS137v9, CMS146v9, CMS153v9, CMS154v9, CMS155v9, CMS159v9.
You'll need to match the measure's numerator UUID and matching strata UUIDs together similar to the previous year: https://github.com/CMSgov/qpp-measures-data/blob/develop/util/measures/2020/additional-stratifications.json

Reply:
whoops, i added this but accidentally removed it in one of the commits lol